UM/Scene/ToolHandle.py

```diff
@@ -25,15 +25,17 @@ class ToolHandle(SceneNode.SceneNode):
     ZAxis = 4
     AllAxis = 5
 
-    DisabledColor = Color(0.5, 0.5, 0.5, 1.0)
```

Comment:
Note that in my opinion these should not be themed. Instead, the display colour and the selection colour should have been split up. Having the selection colour themed makes it much more likely to collide with object colours, since we no longer have precise control over the selection colour.